src/model/comment.ml

```diff
@@ -79,7 +81,7 @@ type attached_block_element = [ nestable_block_element | `Tag of tag ]
 
 type block_element =
   [ nestable_block_element
-  | `Heading of heading_level * Identifier.Label.t * link_content
+  | `Heading of heading_level * Identifier.Label.t * link_content * paragraph_style option
```

Comment:
Do we really want to allow custom alignment on headings ?

Reply:
Indeed not!
@jonludlam, what’s do you say?

Reply:
the intent here is to preserve compatibility with ocamldoc - so does ocamldoc allow this sort of styling on headings?

Reply:
ocamldoc allows alignment on every elements, including code blocks and headings. I don't think we need to do the same thing as ocamldoc here.
I find 4 `.mli` files containing these when grepping the source of every packages on opam (excluding odoc and ocaml) and only one has more than a paragraph in it (https://github.com/savonet/ocaml-duppy/blob/master/src/duppy.mli#L26, which won't work anyway due to the `{i`, which is more restricted).

Reply:
Just kill the feature, this feels like a waste of time. Not only does give a way to hurt UX (which relies on rendering consistency) but there are tons of other small issues in the markup worth solving which can drastically improve UX.

The particular example given by @Julow is not even about alignement it's about quoting (e.g. like markdown provides).

FWIW I never bothered supporting alignment on the stylesheets I made for odoc and ocamldoc for the past 10 years and no one ever complained despite these being reused in the wild and consumed via `odig`.

Reply:
Fortunately / unfortunately, the fix is almost done! 

> there are tons of other small issues in the markup worth solving which can drastically improve UX.

However, I do agree with you on that.

Reply:
So we leave out customising heading alignment, right?

Reply:
> Fortunately / unfortunately, the fix is almost done!

I understand you have put hard work into this and I'll let the official maintainer @jonludlam judge what is best. 

But I just note the fix adds 160 loc to the code base for a dubious feature that @Julow has proved to be publicly almost universally unused (and that I pledge I won't support in `odig`'s themes). Personally I think we should exercice a bit of good jugement on the "feature parity" with `ocamldoc`. There are (very few) things `ocamldoc` didn't get right and there's no need to perpetuate them.

Reply:
If this is not merged, I think we need to accept these elements in the parser for compatibility (which would print a warning and won't apply the styling).

Reply:
Agreed with adding them to the parser - we can at least then remove the markup from the output. @lubegasimon can you please close this PR and open a new one with just the parser changes -- hopefully the rebase after the recent parser changes won't be too annoying. Let's ask for opinions at the next dev meeting and we can come to a collective decision -- I have to admit I wasn't aware it was quite so infrequently used, so that's very helpful data @Julow.